TimeTrackerClient/TimeTrackerClient/Views/Login/LoginFormView.swift

```diff
@@ -27,36 +29,62 @@ struct LoginFormView: View {
                     .foregroundColor(.red)
                     .font(Font.custom("Avenir-Light", size: 20))
                     .padding()
+
                 HStack {
                     Image(systemName: "person.fill")
-                    TextField("", text: $viewModel.username)
-                        .placeholder(when: viewModel.username.isEmpty) {
-                            Text("E-mail").foregroundColor(.cGray)
-                        }
-                        .cornerRadius(10)
-                        .foregroundColor(.cBlack)
-                        .autocapitalization(.none)
-                        .disableAutocorrection(true)
-                        .keyboardType(.emailAddress)
-                        .onTapGesture {
-                            viewModel.errrorMessage = ""
-                        }
-                }.underlineTextField()
+                    CustomTextField(
```

Comment:
-[] facut rollback la `TextField`
-[] folosit `@FocusState` si va functiona pt `iOS 15 +` only [reference](https://www.hackingwithswift.com/quick-start/swiftui/how-to-dismiss-the-keyboard-for-a-textfield)